lib/declare_schema/model/field_spec.rb

```diff
@@ -54,6 +53,9 @@ def initialize(model, name, type, options = {})
           end
         when :string
           @options[:limit] or raise "limit must be given for :string field #{model}##{@name}: #{@options.inspect}; do you want `limit: 255`?"
+        when :bigint
+          @type = :integer
+          @options[:limit] = 8
```

Comment:
Beware of mutating the `options` passed in from the caller--they might not like that! I don't see a `dup` above, so a good way to get a copy is with `merge`:
```
@options = options.merge(limit: 8)
```